Decred Wallet/Features/Navigation Menu/NavigationMenuViewController.swift

```diff
@@ -183,52 +183,51 @@ extension NavigationMenuViewController: SyncProgressListenerProtocol {
         self.stopRefreshingBestBlockAge()
         
         self.resetSyncViews()
-        self.syncStatusLabel.text = "Restarting sync..."
+        self.syncStatusLabel.text = LocalizedStrings.restartingSync
     }
     
     func onStarted(_ wasRestarted: Bool) {
-        self.syncStatusLabel.text = wasRestarted ? "Restarting sync..." : "Connecting to peers."
+        self.syncStatusLabel.text = wasRestarted ? LocalizedStrings.restartingSync : LocalizedStrings.connectingToPeers
     }
     
     func onPeerConnectedOrDisconnected(_ numberOfConnectedPeers: Int32) {
         if AppDelegate.walletLoader.isSynced {
-            self.syncStatusLabel.text = "Synced with \(AppDelegate.walletLoader.syncer.connectedPeers)"
+            self.syncStatusLabel.text = String(format: LocalizedStrings.syncedWith, AppDelegate.walletLoader.syncer.connectedPeers)
         }
     }
     
     func onHeadersFetchProgress(_ progressReport: DcrlibwalletHeadersFetchProgressReport) {
         self.handleGeneralProgressReport(progressReport.generalSyncProgress!)
         
-        self.syncStatusLabel.text = "Fetching block headers."
+        self.syncStatusLabel.text = LocalizedStrings.fetchingBlockHeaders
         if progressReport.currentHeaderTimestamp != 0 {
-            self.bestBlockLabel.text = "\(progressReport.totalHeadersToFetch - progressReport.fetchedHeadersCount) blocks behind."
+            self.bestBlockLabel.text = String(format: LocalizedStrings.blocksBehind, progressReport.totalHeadersToFetch - progressReport.fetchedHeadersCount)
             if progressReport.bestBlockAge != "" {
-                self.bestBlockAgeLabel.text = "\(progressReport.bestBlockAge) ago"
+                self.bestBlockAgeLabel.text = String(format: LocalizedStrings.bestBlockAgeAgo, progressReport.bestBlockAge)
             }
         }
     }
     
     func onAddressDiscoveryProgress(_ progressReport: DcrlibwalletAddressDiscoveryProgressReport) {
         self.handleGeneralProgressReport(progressReport.generalSyncProgress!)
         
-        self.syncStatusLabel.text = "Discovering used addresses."
-        self.bestBlockLabel.text = "\(progressReport.generalSyncProgress!.totalSyncProgress)% completed, \(progressReport.generalSyncProgress!.totalTimeRemaining) left."
+        self.syncStatusLabel.text = LocalizedStrings.discoveringUsedAddresses
+        self.bestBlockLabel.text = String(format: LocalizedStrings.generalSyncProgressCompletedleft, progressReport.generalSyncProgress!.totalSyncProgress,progressReport.generalSyncProgress!.totalTimeRemaining)
```

Comment:
There's a comma in this line without a succeeding space (space after the comma). There should be a single space after each comma to separate the arguments in the function.